addons/sale/views/sale_portal_templates.xml

```diff
@@ -398,100 +398,118 @@
-                        id="shipping_address"
-                        class="col-lg-6">
-                        <strong class="d-block mb-1">
-                            Shipping Address:
-                        </strong>
-                        <address t-field="sale_order.partner_shipping_id"
-                            t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": True}'/>
                     </div>
+                </span>
+                <!-- Information -->
+                <div id="sale_info" class="col-12 col-lg-5 mb-3 mb-lg-0">
+                    <span id="sale_info_title">
+                        <h5 class="mb-1">Sale Information</h5>
+                        <hr class="my-0"/>
+                    </span>
+                    <table class="table table-borderless table-sm">
+                        <tbody style="white-space:nowrap" id="sale_info_table">
+                            <tr>
+                                <th t-if="sale_order.state in ['sale', 'cancel']" class="text-end pb-0">Order Date:</th>
+                                <th t-else="" class="text-end pb-0">Quotation Date:</th>
+                                <td class="w-100 pb-0 text-wrap"><span t-field="sale_order.date_order" t-options='{"widget": "date"}'/></td>
+                            </tr>
+                            <tr t-if="sale_order.validity_date">
+                                <th class="text-end pb-0">Expiration Date:</th>
+                                <td class="w-100 pb-0 text-wrap"><span t-field="sale_order.validity_date" t-options='{"widget": "date"}'/></td>
+                            </tr>
+                            <tr t-if="sale_order.client_order_ref">
+                                <th class="text-end pb-0">Your Reference:</th>
+                                <td class="w-100 pb-0 text-wrap"><span t-field="sale_order.client_order_ref"/></td>
+                            </tr>
+                        </tbody>
+                    </table>
                 </div>
-                <div class="row" t-if="sale_order.client_order_ref">
-                    <div class="col-lg-6">
-                        <strong class="d-block mb-1">Your Reference:</strong>
-                        <t t-out="sale_order.client_order_ref"/>
-                    </div>
+
+                <!-- ======  Customer Information  ====== -->
+                <div id="customer_info" class="col-12 col-lg-5 offset-lg-1">
+                    <h5 class="mb-1">
+                        <t t-if="sale_order.partner_shipping_id == sale_order.partner_invoice_id">
+                            Invoicing and Shipping Address
+                        </t>
+                        <t t-else="">
+                            Invoicing Address
+                        </t>
+                        <small>
+                            <a t-if="sale_order.partner_id == sale_order.partner_id"
```

Comment:
Why isn't the t-if on the small ? (maybe for sub?)
Also, is the `<small` + `class="small"` really necessary ?

Reply:
:+1: for t-if , double small is to make if extra-smol (and changing the heading level result in newline)